### DISCUSSION TITLE
仅单显示器正常，多显示器不正常

### DISCUSSION THREAD

**Discussion by msglog** [Planshit/Tai#275] [Q&A]
仅单显示器正常，多显示器不正常。

笔记本自带显示器打游戏，外接了一个显示器放在旁边看视频（扩展的模式，将视频软件拖到副屏）

**Comment by noberumotto** [comment]
理论上仅记录当前焦点屏幕上的应用

**Comment by msglog** [comment]
> 理论上仅记录当前焦点屏幕上的应用

但是多屏幕时，副屏内显示的程序是全屏撑满的情况，这种副屏所使用的程序的使用时间，理应也计入监控程序内的计算吧？

> 理论上仅记录当前焦点屏幕上的应用

**Comment by noberumotto** [comment]
不在计算范围内的，因为鼠标焦点不在。可以试试关联功能，就能同时记录了
